students/jemielity_kamil/lesson_02_flow_control/the_second_maximum.py

```diff
@@ -0,0 +1,16 @@
+
+maximum = 0
+second_maximum = 0
+
+while True:
+    n = int(input("Integer: "))
+    if n == 0:
```

Comment:
Usually in python we use convention:
'if not n' instead of 'n == 0'. It's more pythonic but on the other side it's also more generic, n might be in [0, None, '', ""]